gpcontrib/gpcloud/src/s3interface.cpp

```diff
@@ -239,8 +239,13 @@ bool S3InterfaceService::parseBucketXML(ListBucketResult *result, xmlParserCtxtP
                 }
                 if (!xmlStrcmp(contNode->name, (const xmlChar *)"Size")) {
                     key_size = (char *)xmlNodeGetContent(contNode);
-                    // Size of S3 file is a natural number, don't worry
-                    size = (uint64_t)atoll((const char *)key_size);
+
+					if (key_size) {
```

Comment:
according to ORCA style
```suggestion
					if (NULL != key_size) {
```

Reply:
Well, it's not orca and it has different style
https://github.com/arenadata/gpdb/blob/adb-6.x-dev/gpcontrib/gpcloud/src/s3interface.cpp#L256-L259
I just tried to keep it in line